modules/web-console/frontend/app/services/ErrorParser.spec.ts

```diff
@@ -20,13 +20,31 @@ import JavaTypes from './JavaTypes.service';
 const parser = new ErrorParser(new JavaTypes());
 
 import { assert } from 'chai';
+import foreach from 'lodash/forEach'
```

Comment:
Why not use `Array#forEach` instead? I have nothing against Lodash, but this looks like an overkill to me.